docs/conf.py

```diff
@@ -87,9 +87,16 @@
     "rtd-dev": ("https://dev.readthedocs.io/en/latest/", None),
     "jupyter": ("https://docs.jupyter.org/en/latest/", None),
 }
-# Redundant in Sphinx 5.0
+
+# Intersphinx: Do not try to resolve unresolved labels that aren't explicitly prefixed.
+# The default setting for intersphinx_disabled_reftypes can cause some pretty bad
+# breakage because we have rtd and rtd-dev stable versions in our mappings.
+# Hence, if we refactor labels, we won't see broken references, since the
+# currently active stable mapping keeps resolving.
+# Recommending doing this on all projects with Intersphinx.
```

Comment:
Not sure what this line means, but doesn't seem like this is the right place for it?

Reply:
Removed